src/scss/components/_thumbnail.scss

```diff
@@ -10,22 +10,22 @@
 **/
 
 .thumbnail {
```

Comment:
@iondrimba yes! He respect this rule and warns about alphabetical order on mixins too 😄